manifests/w/WinFsp/WinFsp/1.9.21096/WinFsp.WinFsp.yaml

```
@@ -10,10 +10,11 @@ Tags:
 ShortDescription: Windows File System Proxy - FUSE for Windows
 PackageUrl: https://github.com/billziss-gh/winfsp
 Installers:
-- Architecture: x86
+- Architecture: x64
```

Sorry, looks like this installer supports both x86 and x64, so we should specify x86 for flexibility

```suggestion
- Architecture: x86
```